app/models/notification.rb

```diff
@@ -1,5 +1,7 @@
 class Notification < ActiveRecord::Base
 #  serialize :departments, Array
+  has_paper_trail
```

Comment:
i don't see `gem 'paper_trail'` in the `Gemfile`  or this PR so i don't see how it will work.

also i don't see what this has to do with the trim label.